fitbenchmarking/fitbenchmarking
# use mathjax for maths in results pages

#### Description of Work

Fixes #683 -- instead of looking like this:
![image](https://user-images.githubusercontent.com/13645545/100432390-74a08480-3091-11eb-82e0-1e4634f896cc.png)
equations should now look like:
![image](https://user-images.githubusercontent.com/13645545/100432312-589ce300-3091-11eb-853e-bcb35a03966f.png)



#### Testing Instructions

1.  Run fitbenchmarking, ensure the equations in the results pages look good
2.  Run fitbenchmarking without being connected to the internet -- do they look the same?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
